src/main.jsx

```diff
@@ -1,15 +1,25 @@
 import React from 'react';
 import { RecoilRoot } from 'recoil';
 import ReactDOM from 'react-dom/client';
-import sync from './sync';
 
 import App from './components/App';
 import './main.css';
 import store from './store';
 
-const { RecoilSyncRoot } = sync;
+import fsSync from './sync/fs';
 
-store.illustrations.refresh();
+// Start FS Syncing
+fsSync();
+
+// store.illustrations.reset();
+// store.illustrations.refresh();
+//   .then(() => store.illustrations.get()).then(console.log);
+
+document.addEventListener('keydown', (event) => {
+  if (event.isComposing || event.key === ' ') {
+    store.illustrations.get().then(console.log);
+  }
+});
 
 const SuspenseTest = function SuspenseTest() {
```

Comment:
Lets figure out what to do with this. If were keeping it as “loading…” it should at least be its own component.